packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -24,18 +24,45 @@
   @include mdc-ripple-radius-bounded;
   @include mdc-chip-corner-radius($mdc-chip-border-radius-default);
   @include mdc-chip-fill-color($mdc-chip-fill-color-default);
-  @include mdc-chip-ink-color($mdc-chip-ink-color-default);
+  @include mdc-chip-ink-color(text-primary-on-light);
 
-  display: inline-block;
-  margin: 5px;
+  display: inline-flex;
+  align-items: center;
   padding: $mdc-chip-vertical-padding $mdc-chip-horizontal-padding;
   cursor: pointer;
+  outline: none;
   overflow: hidden;
 }
 
 .mdc-chip__text {
-  display: inline-block;
-  vertical-align: middle;
   // Set line-height to be <18px to force the content height of mdc-chip to be 18px.
   line-height: 17px;
 }
+
+.mdc-chip__icon {
+  color: $mdc-chip-icon-color;
+  vertical-align: middle;
+  border-radius: 50%;
+  outline: none;
+}
+
+.mdc-chip__icon--leading {
+  margin: -4px 4px -4px -4px;
+}
+
+.mdc-chip__icon--trailing {
+  margin: 0 -4px 0 4px;
+  &:hover {
+    color: $mdc-chip-trailing-icon-hover-color;
+  }
+
+  &:focus {
+    color: $mdc-chip-trailing-icon-focus-color;
+  }
+}
+
+.mdc-chip__icon.mdc-chip__icon--trailing {
```

Comment:
Add a comment indicating these are more specific to ensure they override icon styles in other libraries e.g. material-icons (like in https://github.com/material-components/material-components-web/pull/2242/files)